### PR TITLE
[fix] cdc data timestamp now drawn from source 

### PR DESCRIPTION
**Pull request by BriceWolfgang**

### CODE REVIEW

File: covid/extract.py [hunk 1]
```diff
@@ -49,21 +49,32 @@ def get_state_abbreviations_to_names():
 
 def power_bi_extractor(response):
     data = json.loads(response.text)
-    timestamp = data['results'][0]['result']['data']['timestamp']
+    timestamp = extract_cdc_data_date()
```
Comment (lucasmbrown) [inline covid/extract.py:52]: Very nice!

**Review by lucasmbrown — APPROVED** [review]
@patricksheehan should approve before we merge to see if this meets his needs, but LGTM!

File: covid/extract.py [hunk 2]
```diff
@@ -49,21 +49,32 @@ def get_state_abbreviations_to_names():
...
     value_list = data['results'][0]['result']['data']['dsr']['DS'][0]['PH'][1]['DM1']
     for vl in value_list:
         yield vl['C'] + [timestamp]
 
 
 # CDC data source: https://www.cdc.gov/nhsn/covid19/report-patient-impact.html
 
+
+def extract_cdc_data_date():
```
Comment (patricksheehan) [inline covid/extract.py:61]: 💯

**Review by patricksheehan — APPROVED** [review]
LGTM, thanks for the quick fix!

**Pull request merged**